src/main/java/com/wenqi/bootwebdemo/controller/JFYController.java

```diff
@@ -0,0 +1,54 @@
+package com.wenqi.bootwebdemo.controller;
+
+import com.mashape.unirest.http.HttpResponse;
+import com.mashape.unirest.http.JsonNode;
+import com.mashape.unirest.http.Unirest;
+import com.wenqi.bootwebdemo.service.JFYService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.util.stream.Collectors.joining;
+
+// Used to call external API
+@RestController
+public class JFYController {
+    @Autowired
+    JFYService jfyService;
+
+    @GetMapping(value = "/jfy")
+    private String getClient(@RequestParam int appid, @RequestParam int scene, @RequestParam int user_id, @RequestParam int pageSize) throws Exception {
```

Comment:
private?

Reply:
Both invokable (tested via Postman). But will unify to public methods for controllers.